Makefile

```
@@ -0,0 +1,15 @@
+# Multi-firm operations — convenience targets.
+# Wrappers around scripts that the pre-commit hook also calls.
+
+.PHONY: validate validate-params validate-data test
+
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)